idearium-lib/test/data/test.csv

```diff
@@ -0,0 +1,6 @@
+col1,col2,col3,col4,col5
+Test1.1,Test1.2,Test1.3,Test1.4,Test1.5
+Test2.1,Test2.2,Test2.3,Test2.4,Test2.5
+Test3.1,Test3.2,Test3.3,Test3.4,Test3.5
+Test4.1,Test4.2,Test4.3,Test4.4,Test4.5
```

Comment:
Maybe this test data needs to be a little more complex? For example, a row with an empty value in one of the columns would be good. Also, perhaps a column with `""`. Perhaps another column with an emoji? Just to make this more complete and fool proof in future uses.